### PR TITLE
Bump Sentry JavasScript 7.119.0 / Wizard, Exposes Replay feature

### PR DESCRIPTION
**Pull request by lucas-zimerman**
This PR exposes the Replay feature that was already working but not exposed.
It also bumps Sentry Wizard and  JavaScript to the latest V7 (V8 contains break changes that may be bumped on a future PR)

No break changes were introduced but some additional fields were exposed to the users.

- Sample Event: https://sentry-sdks.sentry.io/issues/5946875129/events/f8f5bb9285fc4f12b38722af581e92e2/?project=5627302
- Sample Replay: https://sentry-sdks.sentry.io/replays/c7fa3a0e52c74a65b5ea9b7c51e65bd6/?referrer=%2Fissues%2F%3AgroupId%2Fevents%2F%3AeventId%2F&t=0&t_main=errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+    replaysOnErrorSampleRate: 1,
+
+    // Tracing sampling filter.
+    tracesSampleRate: 1,
+  });
+```
+
+### Dependencies
+
+- Bump Sentry JavaScript SDK to `7.118.0` ([#354](https://github.com/getsentry/sentry-cordova/pull/354))
```
Comment (krystofwoldrich) [inline CHANGELOG.md:40]: The latest v7 is `7.119.0`, let's bump it to that (if there are no issue with that for cordova).

File: sample/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "ecosystem:cordova"
   ],
   "devDependencies": {
-    "cordova-android": "^12.0.1",
```
Comment (krystofwoldrich) [inline sample/package.json:23]: These changes seem to be not relevant for the JS upgrade. Likely leaking from the Android SDK bump.

File: src/js/sentry-cordova.ts [hunk 3]
```diff
@@ -1,43 +1,57 @@
...
+  replayIntegration,
+  browserTracingIntegration,
+} from '@sentry/browser';
 
 export type { CordovaOptions } from './options';
 export { init, nativeCrash } from './sdk';
 export { SDK_NAME, SDK_VERSION } from './version';
 
-import * as Integrations from './integrations';
-export { Integrations };
```
Comment (krystofwoldrich) [inline src/js/sentry-cordova.ts:40]: It looks like we don't export the Cordova Integrations anymore.

That would be a breaking change.

File: sample/package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
...
+    "cordova-android": "^13.0.0",
```
Comment (lucas-zimerman) [inline sample/package.json:23]:
```suggestion
    "cordova-android": "^12.0.1",
```

File: src/js/sentry-cordova.ts [hunk 5]
```diff
@@ -1,43 +1,57 @@
...
-export { BrowserIntegrations };
+
+export { Integrations as BrowserIntegrations } from '@sentry/browser';
```
Comment (lucas-zimerman) [inline src/js/sentry-cordova.ts:55]:
```suggestion
export { Integrations as BrowserIntegrations } from '@sentry/browser';

import * as Integrations from './integrations';
export { Integrations };
```

File: src/js/sentry-cordova.ts (hunk 3 above)
Comment (lucas-zimerman) [reply src/js/sentry-cordova.ts:40]: True, thats a abreaking change, thanks for catching this!

File: sample/www/js/index.js [hunk 6]
```diff
@@ -23,10 +23,30 @@ document.addEventListener('deviceready', onDeviceReady, false);
...
+      // Replay integration.
+      Sentry.replayIntegration({
+        maskAllInputs: false,
+        blockAllMedia: true
+      }),
+      // Tracing integration.
+      Sentry.browserTracingIntegration(),
+    ],
+
+
```
Comment (lucas-zimerman) [inline sample/www/js/index.js:43]:
```suggestion
```

File: package.json [hunk 7]
```diff
@@ -24,16 +24,16 @@
   ],
   "description": "Official Sentry SDK for Cordova",
   "dependencies": {
-    "@sentry/browser": "7.34.0",
-    "@sentry/core": "7.34.0",
-    "@sentry/hub": "7.34.0",
-    "@sentry/types": "7.34.0",
-    "@sentry/utils": "7.34.0",
-    "@sentry/wizard": "^3.21.0"
+    "@sentry/browser": "7.119.0",
```
Comment (krystofwoldrich) [inline package.json:27]: Let do one more bump to include the latest security patch. 

https://github.com/getsentry/sentry-javascript/releases/tag/7.119.1

**Review by lucas-zimerman — COMMENTED** [review]
LGTM!

**Pull request merged**